_posts/2021-04-28-fed-talk-s01e02-theme-providers.md

```diff
@@ -634,7 +634,21 @@ export const palette: PaletteOptions = {
   tonalOffset: 0.2,
 }
 ```
+And replace the default palette of MUI with our newly created palette, through our theme file.
 
+```tsx
+// theme/index.tsx
+
+import { createMuiTheme, responsiveFontSizes } from '@material-ui/core';
+
+import { typography } from './typography';
+
+export default responsiveFontSizes(createMuiTheme({
+  typography,
+  palette
+}))
+```
+ 
 As the comments in the above code snippet suggest, you can let MUI calculate the Light/Dark values for you for free, otherwise manually add them for each Primary and Secondary objects.
```

Comment:
> As the comments in the above code snippet suggest, you can let MUI calculate the Light/Dark values for you for free, otherwise manually add them for each Primary and Secondary objects.

This should probably be moved up, since it relates to the code snippet higher up - line 611